src/lib/viewers/doc/PresentationViewer.js

```diff
@@ -129,7 +129,11 @@ class PresentationViewer extends DocBaseViewer {
      * @override
      */
     handleScrollToAnnotation(data) {
-        this.setPage(getProp(data, 'target.location.value', 1));
+        const targetPage = getProp(data, 'target.location.value', 1);
+
+        if (this.pdfViewer.currentPageNumber !== targetPage) {
```

Comment:
Do we need to check if pdfViewer exists?

Reply:
We don't really do that anywhere else, including `setPage`. Seems like overkill?